spec/bolt/node/ssh_spec.rb

```diff
@@ -14,10 +14,9 @@ def mk_config(conf)
   end
 
   let(:hostname) { ENV['BOLT_SSH_HOST'] || "localhost" }
-  let(:user) { ENV['BOLT_SSH_USER'] || "vagrant" }
-  let(:password) { ENV['BOLT_SSH_PASSWORD'] || "vagrant" }
-  let(:port) { ENV['BOLT_SSH_PORT'] || 20022 }
-  let(:key) { ENV['BOLT_SSH_KEY'] || Dir[".vagrant/**/private_key"] }
+  let(:user) { ENV['BOLT_SSH_USER'] || "root" }
```

Comment:
Eventually I can see us wanting to test both
- login as root and sudo to non-root
- login as non-root and sudo to root

I have some concern about changing from non-root to root though, as I believe some tests around sudo specifically test escalating to root.

Reply:
M'k -- root is the default user for containers.  Should adding a new user with escalating privileges be part of this ticket, or should it be separate work?

Reply:
Creating non-default users could be part of the container creation or test. We can write the tests so they require root access, but I'd like to preserve tests that login to a non-root account and test that they can escalate correctly.

If you want to make that separate work, then make sure the current tests still correctly check that run-as works.